atd-vze/src/queries/gqlAbstract.js

```diff
@@ -207,6 +207,10 @@ gqlAbstractTableAggregateName (
    */
   setOrder(key, syntax) {
     if (this.config && this.config["order_by"]) {
+      // First, RESET the order_by value.
+      // - Our assumption is that there should only by 1 order_by at a time.
+      this.config["order_by"] = {};
+      // Now, set new key, syntax pair for order_by
```

Comment:
This turned out to be the solution @mddilley. When updating the sort order, it was possible to create duplicate nested queries using slightly different syntax. 

ex:
```js
crashes_count_cost_summary: "{ est_comp_cost: desc}"
crashes_count_cost_summary: { est_comp_cost }: "desc"
```

![Screen Shot 2020-09-20 at 9 16 57 PM](https://user-images.githubusercontent.com/5697474/93729129-c526cd80-fb88-11ea-8234-2422ab4fe564.jpg)

Reply:
Awesome, thanks for the screenshot and explanation. Tested on my end, and the bug is gone. 👍